### PR TITLE
Added filetype csv2 and csv3 to PCAxis.Web.Controls

### PR DESCRIPTION
**Pull request by masv-scb**

### CODE REVIEW

File: PCAxis.Web.Controls/FileTypes/Csv/Csv2FileType.vb [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+
+''' <summary>
+''' </summary>
+''' <remarks></remarks>
+Partial Public Class Csv2FileType
+    Inherits FileTypeMarkerControlBase(Of Csv2FileTypeCodebehind, Csv2FileType)
+
+    ''' <summary>
+    ''' Creates a html file and sends it to the user
```
Comment (MikaelNordberg) [inline PCAxis.Web.Controls/FileTypes/Csv/Csv2FileType.vb:17]: Should be csv in the comment

File: PCAxis.Web.Controls/FileTypes/Csv/Csv3FileType.vb [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+
+''' <summary>
+''' </summary>
+''' <remarks></remarks>
+Partial Public Class Csv3FileType
+    Inherits FileTypeMarkerControlBase(Of Csv3FileTypeCodebehind, Csv3FileType)
+
+    ''' <summary>
+    ''' Creates a html file and sends it to the user
```
Comment (MikaelNordberg) [inline PCAxis.Web.Controls/FileTypes/Csv/Csv3FileType.vb:17]: Should be csv in the comment

**Review by MikaelNordberg — CHANGES_REQUESTED** [review]
Looks good except for the comments

**Review by MikaelNordberg — APPROVED** [review]
Looks good

**Review by likp — COMMENTED** [review]
@MikaelNordberg should we also be able to override saved queries with csv2 and csv3?

**Review by MikaelNordberg — APPROVED** [review]
Looks good

**Review by MikaelNordberg — APPROVED** [review]
Looks good

**Review by likp — APPROVED** [review]

**Pull request merged**